build-farm/platform-specific-configurations/linux.sh

```diff
@@ -135,6 +135,17 @@ if [ "${VARIANT}" == "${BUILD_VARIANT_DRAGONWELL}" ] && [ "$JAVA_FEATURE_VERSION
     fi
     export "${BOOT_JDK_VARIABLE}"="$PWD/jdk-8"
   fi
+elif [ "${VARIANT}" == "${BUILD_VARIANT_LOONGSON}" ]; then
+  if [ "$(uname -m)" = "loongarch64" ]; then
+    if [ -d /opt/j2sdk-images ]; then
```

Comment:
1. Do not consider the situation `-d /opt/j2sdk-images`
2. Consider the situation in which `$JAVA_FEATURE_VERSION` is not 8. We may add support for more java verison, like 17 and 11.

Reply:
The proposal seems reasonable. I will fix this issue.